src/sparsezoo/v2/model_directory.py

```diff
@@ -404,19 +407,15 @@ def _validate_with_onnx_runtime(self):
             # TODO: This does not work for now, the
             #  outputs are quite different. To be investigated.
             validation.append(
-                all([np.array_equal(x, y) for x, y in zip(target_output, output)])
+                all((np.array_equal(x, y) for x, y in zip(target_output, output)))
```

Comment:
This will work, a small nit for future: You don't need generator braces `()` inside `all`; as in you could do something like:
 `all(np.array_equal(x, y) for x, y in zip(target_output, output))`

Reply:
yummy